### PR TITLE
Added Lap Comparison Tab

### PR DESCRIPTION
**Pull request by elft3r**
Hey @kart7990,
I have added a new Tab with the Lap Comparison table from the previous version.

Let me know if you have additional information you'd like to see here. I already had an idea to add a graph with the two selected times.

Best,
Jochen

### CODE REVIEW

File: src/components/timing/lap-comparison.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                "" + lap.getLapNumber(),
+                "" + lap.getStint(),
+                lap.getLapTime(),
+              ];
+            })
+        : [];
+    };
+
+    setData(getData());
+  }, [selectedDriver, selectedLaps]);
```
Comment (kart7990) [inline src/components/timing/lap-comparison.tsx]: Are these useEffects (and related code) needed here if the LapTime component does it all?

Comment (elft3r) [reply src/components/timing/lap-comparison.tsx]: Thanks for pointing this out. I forgot to remove it after refactoring the code to the final version 🤦

Comment (kart7990) [reply src/components/timing/lap-comparison.tsx]: I was talking about in this `lap-comparison` component.

Comment (kart7990) [reply src/components/timing/lap-comparison.tsx]: Meaning all it needs to do is render the two LapTime components (which it is doing), but it doesn't need all the useEffect logic, or am I looking at it wrong?

Comment (elft3r) [reply src/components/timing/lap-comparison.tsx]: No, you are correct. I don't know why I didn't remove this from there. I removed the `useEffect` code now and added another selector to select the session.

**Review by kart7990 — APPROVED** [review]

**Pull request merged**